shared/src/common/enums/modelName/model-name.enum.ts

```diff
@@ -0,0 +1,11 @@
+enum ModelName {
```

Comment:
Let's do file/folder names only in the `kebab` notation
shared/src/common/enums/model-name/model-name.enum.ts